apps/condo/domains/resident/schema/RegisterResidentService.js

```diff
@@ -48,13 +62,15 @@ const RegisterResidentService = new GQLCustomSchema('RegisterResidentService', {
                     await ResidentAPI.update(context, existingResident.id, {
                         ...attrs,
                         deletedAt: null,
-                        address: undefined,
-                        addressMeta: undefined,
-                        unitName: undefined,
+                        address: undefined, // skip value from attrs
+                        addressMeta: undefined, // skip value from attrs
+                        unitName: undefined, // skip value from attrs
                     })
                     id = existingResident.id
                 } else {
-                    const resident = await Resident.create(context, attrs)
+                    const propertyAddress = removeFlatNum(address, addressMeta.data.flat_type)
```

Comment:
Can a function `getAddressUpToBuildingFrom ` be reused in this case?

Reply:
done